packages/account-v2/src/containers/LoginHistoryTable/LoginHistoryTable.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useMemo } from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Table } from '@deriv-com/ui';
+import { formattedLoginHistoryData } from '../../pages/LoginHistory/formattedLoginHistoryData';
+
+type TLoginHistoryData = {
+    loginHistory: LoginHistory;
+};
+
+const headers = {
+    action: 'Action',
+    browser: 'Browser',
+    datetime: 'Date and Time',
+    ipAddress: 'IP Address',
+    status: 'Status',
+};
+
+const columnOrder = ['datetime', 'action', 'browser', 'ipAddress', 'status'] as const;
+
+export const LoginHistoryTable = ({ loginHistory }: TLoginHistoryData) => {
+    const formattedLoginHistory = useMemo(() => formattedLoginHistoryData(loginHistory), [loginHistory]);
+    const columns = columnOrder.map(key => ({ header: headers[key] }));
```

Comment:
Rather than having columns and headers object, can we merge them into an array of objects with the required objects?

Reply:
I think it's better to keep them separate as we need to keep the column order and if we merge them eslint complains about sort keys

Reply:
Not sure about this, since I implemented the other way around inside constants file and there was no eslint error